### PR TITLE
[feature] add or replace option for view creations

### PR DESCRIPTION
**Pull request by emancu**
`CREATE OR REPLACE VIEW` is similar, but if a view of the same name already exists, it is replaced. The new query must generate the same columns that were generated by the existing view query (that is, the same column names in the same order and with the same data types).

## Test Plan

* [x] Internal tests
* [x] acceptance tests

## References
* https://docs.snowflake.com/en/sql-reference/sql/create-view.html

### CODE REVIEW

File: pkg/resources/view.go [hunk 1]
```diff
@@ -106,10 +116,6 @@ func CreateView(data *schema.ResourceData, meta interface{}) error {
 		builder.WithComment(v.(string))
 	}
 
-	if v, ok := data.GetOk("schema"); ok {
-		builder.WithSchema(v.(string))
-	}
-
```
Comment (emancu) [inline pkg/resources/view.go:112]: The `WithSchema` method was applied in line 104 already.

**Comment by emancu** [comment]
There is something odd with the acceptance-tests, that always fails.
But I noticed that is failing in ALL PRs submitted, so I think it is a bigger problem.

**Comment by ryanking** [comment]
> There is something odd with the acceptance-tests, that always fails.
> But I noticed that is failing in ALL PRs submitted, so I think it is a bigger problem.

Have you gone through the steps in https://github.com/chanzuckerberg/terraform-provider-snowflake#pull-request-ci ?

**Comment by emancu** [comment]
@ryanking I followed the steps and locally (as usual xD) the tests pass with my own account.
However, for this PR I can't make them work.

**Comment by alldoami** [comment]
Ran acceptance tests and passed. Thanks for the work!

**Pull request merged**